src/globals.cpp

```diff
@@ -63,12 +63,14 @@ int BetterSMS::getScreenOrthoWidth() {
     case AspectRatioSetting::WIDE:
         return 853;
     case AspectRatioSetting::ULTRAWIDE:
-        return 1120;
+        return 1137;
```

Comment:
This mismatches the calculation made to get the value `WIDE` returns: `(480 / 9) * 16 == 853` `(480 / 9) * 21 == 1120`

Reply:
There's another way to calculate ultra wide, if you calculate like these:

`(480 * 2560) / 1080`
`480 * (2560 / 1080)`